src/process-tpl.js

```diff
@@ -88,7 +89,13 @@ export default function processTpl(tpl, baseURI) {
 					return genLinkReplaceSymbol(newHref);
 				}
 			}
-
+			/*
+			as font normal
+			*/
+			const asFont = !!match.match(LINK_AS_FONT);
+			if (asFont) {
+				return match;
+			}
 			const preloadOrPrefetchType = match.match(LINK_PRELOAD_OR_PREFETCH_REGEX) && match.match(LINK_HREF_REGEX);
```

Comment:
combine `asFont` condition here, such as
```js
const preloadOrPrefetchType = match.match(LINK_PRELOAD_OR_PREFETCH_REGEX) && match.match(LINK_HREF_REGEX) && !!match.match(LINK_AS_FONT)
```

Reply:
haha 😄, I did this last commit, I'll change back